Resources/Prototypes/Entities/Objects/Tools/access_configurator.yml

```
@@ -59,7 +59,7 @@
       - Theatre
       - Zookeeper #Delta V: Add Zookeeper Access
       - ChiefJustice  #Delta V: Add Chief Justice Access
-      - Prosecutor  #Delta V: Add Prosecutor Access
+      - Inspector  #Delta V: Add Inspector Access
```

yamllint

[warning] missing starting space in comment

(comments)